cypress/integration/bundled-osd/check_advanced_settings.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
I believe we need the `Copyright OpenSearch Contributors` line for new files: https://github.com/opensearch-project/.github/issues/21

Reply:
will do